docs/40_smart-contract-guides/20_deploy-issue-and-transfer-tokens.md

```diff
@@ -58,9 +58,9 @@ warning: transaction executed locally, but may not be confirmed by the network y
 
 ## Step 5: Create the Token
 
-To create a new token, call `create` action with the correct parameters. This action accepts 1 argument, it consists of:
+To create a new token, call `create` action with the correct parameters specified as a json array:
```

Comment:
Suggestion:

call \`create\` action with the correct parameters specified as a json array:
-->
call the \`create\` action with the following parameters specified as a JSON array: